packages/mds-utils/exceptions.ts

```diff
@@ -1,11 +1,11 @@
-class BaseError extends Error {
+export class BaseError extends Error {
```

Comment:
`mds-utils` is sufficiently low-level that you should be able to remove the `@hapi/joi` dependency and leave validation error here. There are reasons we might want to throw `ValidationError` not related to schema validation.